api/src/main/resources/liquibase.xml

```diff
@@ -6,6 +6,17 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 
     <changeSet author="SolDevelo" id="org.openmrs.module.atomfeed-2017-11-08-10:22">
+    	<validCheckSum>3:ce20ea9f79b936f10010d668ef669ca7</validCheckSum>
+        <preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="event_records" /></not>
```

Comment:
Since the change is in just one table, why would the precondition include others?